spec/index.html

```diff
@@ -252,10 +254,10 @@ <h3>Object Lists</h3>
     <p>
       As with predicates often objects are repeated with the same subject and predicate.
       The <a href="#grammar-production-objectList">objectList production</a>
-      matches a series of objects separated by '<code>,</code>' following a predicate.
+      matches a series of objects separated by <code>,</code> following a predicate.
       This expresses a series of RDF Triples with the corresponding subject and predicate
```

Comment:
I think "corresponding" is the wrong word here, but I can't quickly come up with the right one... Perhaps later, or someone else will.

Reply:
```suggestion
      This expresses a series of RDF Triples with the same subject and predicate
```